### PR TITLE
Improve ThoughtSpot extractor documentation.

### PR DESCRIPTION
**Pull request by elic-eon**
### 🤔 Why?

We need ThoughtSpot Everywhere Edition to run out connector.

### 🤓 What?
Update the readme of ThoughtSpot.

### 🧪 Tested?

n/a

### CODE REVIEW

File: metaphor/thought_spot/README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # ThoughtSpot
 
-This connector extracts technical metadata from ThoughtSpot using [ThoughtSpot API](https://try-everywhere.thoughtspot.cloud/v2/#/everywhere/api/rest/playgroundV2)
+This connector extracts technical metadata from ThoughtSpot using [ThoughtSpot REST API v2](https://try-everywhere.thoughtspot.cloud/v2/#/everywhere/api/rest/playgroundV2), which will need the ThoughtSpot Everywhere Edition license. See this documentation [Difference between REST v1 and v2 APIs](https://try-everywhere.thoughtspot.cloud/v2/#/everywhere/documentation/en/?pageid=v1v2-comparison) for more detail.
```
Comment (mars-lan) [inline metaphor/thought_spot/README.md:3]:
```suggestion
This connector extracts technical metadata from ThoughtSpot using [ThoughtSpot REST API v2](https://try-everywhere.thoughtspot.cloud/v2/#/everywhere/api/rest/playgroundV2), which will need the [ThoughtSpot Everywhere](https://www.thoughtspot.com/everywhere) Edition license. See this documentation [Difference between REST v1 and v2 APIs](https://try-everywhere.thoughtspot.cloud/v2/#/everywhere/documentation/en/?pageid=v1v2-comparison) for more detail.
```

Comment (mars-lan) [reply metaphor/thought_spot/README.md:3]: Actually on second thought, let's move the requirement to under "Setup" instead?

Comment (elic-eon) [reply metaphor/thought_spot/README.md]: Cool. It's better to put it under "Setup".

File: metaphor/thought_spot/README.md [hunk 2]
```diff
@@ -1,9 +1,11 @@
 # ThoughtSpot
 
-This connector extracts technical metadata from ThoughtSpot using [ThoughtSpot REST API v2](https://try-everywhere.thoughtspot.cloud/v2/#/everywhere/api/rest/playgroundV2), which will need the [ThoughtSpot Everywhere](https://www.thoughtspot.com/everywhere) Edition license. See this documentation [Difference between REST v1 and v2 APIs](https://try-everywhere.thoughtspot.cloud/v2/#/everywhere/documentation/en/?pageid=v1v2-comparison) for more detail.
+This connector extracts technical metadata from ThoughtSpot using [ThoughtSpot REST API v2](https://try-everywhere.thoughtspot.cloud/v2/#/everywhere/api/rest/playgroundV2).
 
 ## Setup
 
+This connector requires a [ThoughtSpot Everywhere](https://www.thoughtspot.com/everywhere) Edition license to use ThoughtSpot REST API v2. See this documentation [Difference between REST v2 and v2 APIs](https://try-everywhere.thoughtspot.cloud/v2/#/everywhere/documentation/en/?pageid=v1v2-comparison) for more detail.
```
Comment (mars-lan) [inline metaphor/thought_spot/README.md:7]:
```suggestion
This connector requires a [ThoughtSpot Everywhere](https://www.thoughtspot.com/everywhere) Edition license to use ThoughtSpot REST API v2. See this documentation [Difference between REST v2 and v2 APIs](https://try-everywhere.thoughtspot.cloud/v2/#/everywhere/documentation/en/?pageid=v1v2-comparison) for more details.
```

**Pull request merged**